### PR TITLE
Implement preset

### PR DESCRIPTION
**Pull request by novemberborn**
For https://github.com/avajs/ava/issues/1195.

### CODE REVIEW

**Comment by novemberborn** [comment]
This removes the deprecated APIs: https://github.com/avajs/ava/blob/a380fe4f5ffa1d334a2a3786215d3c1ff10a9898/lib/enhance-assert.js#L17:L21

**Review by sindresorhus — APPROVED** [review]

**Comment by novemberborn** [comment]
Forgot that we needed a way to disable power-assert…

File: index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const ESPOWER_PATTERNS = require('./espower-patterns.json');
+
+function buildPreset(context, options) {
+	const plugins = [];
+	if (!options || options.powerAssert !== false) {
```
Comment (sindresorhus) [inline index.js:6]: The option should be documented in the readme.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const ESPOWER_PATTERNS = require('./espower-patterns.json');
```
Comment (sindresorhus) [inline index.js:2]: Add `'use strict';` above here.

**Pull request merged**